apiserver/facades/client/applicationoffers/applicationoffers.go

```diff
@@ -325,11 +318,8 @@ func (api *OffersAPIv5) changeOfferAccess(
 			},
 			Access: accessLevel,
 		},
-		AddUser:  addUser,
-		External: external,
-		ApiUser:  apiUserName,
```

Comment:
issue: The facade now needs to check that the apiUserName has permission sufficient to change the permissions. They need SuperUser across the controller or Admin for the entity.

Reply:
This is already done [here](https://github.com/juju/juju/blob/3f90ecd971597c81ba5c9bc69c17858f1b7ae418/apiserver/facades/client/applicationoffers/applicationoffers.go#L274).